validator/gulpjs/sample/gulpfile.js

```diff
@@ -0,0 +1,18 @@
+const gulp = require('gulp');
```

Comment:
Ah, I was trying to merge this but I can't because the presubmit is still acting up. Please put the license stuff into this file to make it happy.

[06:36:49] Using gulpfile ~/build/ampproject/amphtml/gulpfile.js
[06:36:49] Starting 'presubmit'...
[06:36:56] Did not find required: "Copyright 20(15|16) The AMP HTML Authors\." in validator/gulpjs/sample/gulpfile.js
[06:36:56] ==========
[06:36:56] Did not find required: "Licensed under the Apache License, Version 2\.0" in validator/gulpjs/sample/gulpfile.js
[06:36:56] ==========
[06:36:56] Did not find required: "http\://www\.apache\.org/licenses/LICENSE-2\.0" in validator/gulpjs/sample/gulpfile.js
[06:36:56] ==========
[06:36:58] Adding these terms (e.g. by adding a required LICENSE to the file)
pr-check.js - exiting due to failing command: node_modules/gulp/bin/gulp.js presubmit